src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/DiagnosticPortTestsHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Extensions.Hosting;
+using System.Collections.Generic;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.UnitTests
+{
+    internal static class DiagnosticPortTestsHelper
+    {
+        public static IEnumerable<object[]> GetFileNamesAndEnvironmentVariables()
```

Comment:
Move this to be a method on the ConfigurationTests class. I don't think any other test is going to use this.

Reply:
Maybe rename to `GetConnectionModeTestArguments`